Formula/kakoune.rb

```diff
@@ -21,19 +21,20 @@ class Kakoune < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "606496e883548440a2813b00e3639212da5d3aa11a52d4fd9ae180d373197fc5"
   end
 
-  depends_on macos: :high_sierra # needs C++17
-  depends_on "ncurses"
-
-  uses_from_macos "libxslt" => :build
-
   on_linux do
     depends_on "binutils" => :build
-    depends_on "linux-headers@5.15" => :build
-    depends_on "pkg-config" => :build
   end
 
-  fails_with gcc: "5"
-  fails_with gcc: "6"
+  fails_with :clang do
+    build 1200
```

Comment:
That seems newer than what the README requires for clang: "clang >= 11"

Reply:
I should probably have mentioned this in the PR description.

My assumption, based on other formulas (for example, [1](https://github.com/Homebrew/homebrew-core/blob/f8ba616a4ef7661ff2116414bc1973dce49279f5/Formula/vte3.rb#L44-L47) [2](https://github.com/Homebrew/homebrew-core/blob/f8ba616a4ef7661ff2116414bc1973dce49279f5/Formula/mold.rb#L33-L36) [3](https://github.com/Homebrew/homebrew-core/blob/f8ba616a4ef7661ff2116414bc1973dce49279f5/Formula/grokj2k.rb#L42-L45)), is that this refers to the Xcode clang build number, and not the LLVM version number (which is what the Kakoune README is referring to).

Specifically, 1200 maps to LLVM 10.0.0, and the first Xcode clang build number that maps to LLVM >= 11 is 1205 ([source](https://en.wikipedia.org/wiki/Xcode#Xcode_11.0_-_14.x_(since_SwiftUI_framework))).

My experiments corroborate this - I tried building Kakoune this on macOS 10.15 with clang build 1200, and I get a bunch of errors.

Reply:
I also added 3b48e33e89fe99f304687a8a640ecb32b93fbca1, which adds a build dependency on LLVM in case the system clang version is too old. It is the same strategy that is used in the example formulas linked above.